tests/flow/test_procedures.py

```diff
@@ -319,3 +319,12 @@ def test_procedure_lookup(self):
             # This should not cause an error
             self.env.assertFalse(1)
             pass
+
+    def test_procedure_get_all_procedures(self):
+        actual_resultset = redis_graph.call_procedure("dbms.procedures").result_set
+
+        # The following two procedure are a part of the expected results
+        expected_result = ["db.labels", "READ"]
+        self.env.assertContains(expected_result, actual_resultset)
+        expected_result = ["db.idx.fulltext.createNodeIndex", "WRITE"]
+        self.env.assertContains(expected_result, actual_resultset)
```

Comment:
can you make it a single array?

Reply:
I don't think so, because the assertContains utility works only with searching a singleton in a set